.gitattributes

```diff
@@ -1 +1,2 @@
 src/pixelator/report/webreport/template.html linguist-vendored
+*.png filter=lfs diff=lfs merge=lfs -text
```

Comment:
Could we make this specific for the mpl tests and not all pngs?
Things like rendering the README on github with png's in it could break down otherwise.

Reply:
Good point! done.